apps/meteor/tests/end-to-end/apps/video-conferences.ts

```diff
@@ -284,64 +335,224 @@ describe('Apps - Video Conferences', () => {
 		});
 
 		describe('[/video-conference.info]', () => {
-			let callId: string | undefined;
+			describe('[basic provider]', () => {
```

Comment:
Basic means no support for persistent chat, right? If so, I think you can explicitly put that.